Makefile

```diff
@@ -144,10 +144,19 @@ clean-target-dir:
 
 .venv/bin/python: Makefile
 	rm -rf .venv
+
 	@# --copies is necessary because OS X make checks the mtime of the symlink
 	@# target (/usr/local/bin/python), which is always much older than the
 	@# Makefile, and then proceeds to unconditionally rebuild the venv.
 	$$RELAY_PYTHON_VERSION -m venv --copies .venv
+	.venv/bin/pip install -U pip wheel
+
+	@# Work around https://github.com/confluentinc/confluent-kafka-python/issues/1190
+	if [ "$$(uname -sm)" = "Darwin arm64" ]; then \
+		echo "Using 'librdkafka' from homebrew to build confluent-kafka"; \
+		export C_INCLUDE_PATH="$$(brew --prefix librdkafka)/include"; \
+		export LIBRARY_PATH="$$(brew --prefix librdkafka)/lib"; \
```

Comment:
In my original PR I only needed CPATH.

It seems you used this solution:
https://github.com/confluentinc/confluent-kafka-python/issues/1190#issuecomment-1017410984

Did you happen to find out why CPATH was not good enough? (Just out of curiosity; no need to research it if it's unknown)